utils.go

```diff
@@ -17,17 +17,13 @@ package mysql
 import (
 	"fmt"
 
-	//
+	// apply mysql driver.
 	_ "github.com/go-sql-driver/mysql"
 	"github.com/jmoiron/sqlx"
 )
 
-func connect(config Config) (*sqlx.DB, error) {
-	dataSourceName := fmt.Sprintf(
-		"%s:%s@tcp(%s:%d)/%s",
-		config.User, config.Password, config.Host, config.Port, config.Database,
-	)
-	db, err := sqlx.Open("mysql", dataSourceName)
+func connect(url string) (*sqlx.DB, error) {
```

Comment:
looks like we could get rid of this function (file) altogether and call `sqlx.Open` directly on https://github.com/conduitio-labs/conduit-connector-mysql/pull/10/files#diff-a4d006ab5233eeae48e5dc55c19a888b1a1ab4edcda548ba466e8a88909ed063R74.